docs/cli/orchestration/order-of-execution.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Order of Execution
+description: Define the order stacks are executed in.
+outline: [2, 3]
```

Comment:
Shall we add this as default settings to the config? https://vitepress.dev/reference/default-theme-config#outline

Reply:
does not matter fmpov.